### ISSUE TITLE
Standardize/unify icons across the app

### ISSUE THREAD

**Issue by d4f5409d** [organicmaps/organicmaps#4675]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you would like**
Choosing an open source icon pack and using those particular icons in the entire app.

**Describe alternatives you have considered**
- How do you solve this issue now with Organic Maps or other apps?
Use providers for example like
- https://remixicon.c
- https://feathericons.com/ or
- https://heroicons.com/
- and https://evil-icons.io/ for the iOS variant

**Additional context**
-

**Comment by Jean-BaptisteC** [comment]
Can you give more details ?
On Android, we use native icons (somes icons in svg has need to migrate to xml) and for icons in map, icons is design by community (icons used on Android, IOS and Desktop)

**Comment by d4f5409d** [comment]
I meant the UI only, not the map :)

**Comment by biodranik** [comment]
@d4f5409d it would be helpful if you show some examples of what should be changed into what, so everyone can see and evaluate it )

**Comment by d4f5409d** [comment]
![Screenshot_2023-07-28-15-20-58-88_97294d061ac208a4e8e283a3feb896c9.jpg](https://github.com/organicmaps/organicmaps/assets/76624594/acb689f1-54c4-4fca-b81c-5c0b27570d1d)

I will use the Mastodon app as an example, I mean these type of icons, that have the same color and style.

**Comment by RedAuburn** [comment]
Do you mean on iOS? Android icons are all the same style

**Comment by d4f5409d** [comment]
> Do you mean on iOS? Android icons are all the same style

No. I mean Android and not everywhere have the icond the same style as you told it. For example the about page (social networks)

**Comment by Jean-BaptisteC** [comment]
On about page, we prefer to keep the original icon from social networks.
I have started rework on Android to replace as many possible custom icons with similar Material Icons.
